README.md

```diff
@@ -46,29 +46,33 @@ This removes the immediate connection between human emotions and version numbers
 
 ### Commit message format
 
-**semantic-release** uses the commit messages to determine the type of changes in the codebase. Following formalized conventions for commit messages, **semantic-release** automatically determines the next [semantic version](https://semver.org) number, generates a changelog and publishes the release.
+**semantic-release** uses the commit messages to determine the consumer impact of changes in the codebase.
```

Comment:
i've started breaking lines after sentences. i think this approach is easier to parse than long lines for each paragraph, but isn't as extreme as a max-length requirement. open to feedback